### PR TITLE
rootless: exec join the user+mount namespace

### PR DESCRIPTION
**Pull request by giuseppe**
it is not enough to join the user namespace where the container is
running.  We also need to join the mount namespace so that we can
correctly look-up inside of the container rootfs.  This is necessary
to lookup the mounted /etc/passwd file when --user is specified.

Closes: https://github.com/containers/libpod/issues/2566

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
LGTM
Test unhappiness doesn't appear to be related to these changes.

File: cmd/podman/create.go [hunk 1]
```diff
@@ -884,17 +884,16 @@ func joinOrCreateRootlessUserNamespace(createConfig *cc.CreateConfig, runtime *l
...
 			if err != nil {
-				return false, -1, err
+				return false, -1, errors.Wrapf(err, "cannot read conmon PID file %q", ctr.Config().ConmonPidFile)
 			}
-			if pid == 0 {
-				if createConfig.Pod != "" {
-					continue
-				}
-				return false, -1, errors.Errorf("dependency container %s is not running", ctr.ID())
+			conmonPid, err := strconv.Atoi(string(data))
```
Comment (mheon) [inline cmd/podman/create.go]: Did we drop the `pod == 0` check here, or is it handled in `JoinDirectUserAndMountNS()`

Comment (giuseppe) [reply cmd/podman/create.go]: it should fail as it is not running, but I am going to add it again as it is clearer to understand.

Thanks to catch it

**Comment by MarcoFalke** [comment]
Sorry if this is the wrong place to ask, but is there a release schedule?

I presume when everything in https://github.com/containers/libpod/pulls?q=is%3Apr+is%3Aopen+label%3A%22Release+Notes+1.2.0%22 is merged, this will make it into a release?

**Comment by mheon** [comment]
@MarcoFalke no explicit schedule so far, aside from a general goal of making at least one release a month.

I think we're looking to cut a 1.2 late next week or early the week after, once we land some more work from @baude related to healthchecks

**Comment by rhatdan** [comment]
@MarcoFalke We usually release when major next features get to complete.  Or when there is enough bug fixes to warrant it.  Usually no less then every  other month.

But we don't have a schedule.

**Comment by debarshiray** [comment]
This might have caused #2673

**Pull request merged**